react/src/pages/EnvironmentPage.tsx

```diff
@@ -47,36 +63,34 @@ const EnvironmentPage = () => {
     >
       <Flex
         style={{
-          display: curTabKey === 'imageList' ? 'block' : 'none',
+          display: curTabKey === 'image' ? 'block' : 'none',
           paddingTop: token.paddingContentVerticalSM,
         }}
       >
         {/* @ts-ignore */}
-        <backend-ai-environment-list active={curTabKey === 'imageList'} />
+        <backend-ai-environment-list active={curTabKey === 'image'} />
       </Flex>
       <Flex
         style={{
-          display: curTabKey === 'presetList' ? 'block' : 'none',
+          display: curTabKey === 'preset' ? 'block' : 'none',
           paddingTop: token.paddingContentVerticalSM,
         }}
       >
         {/* @ts-ignore */}
-        <backend-ai-resource-preset-list active={curTabKey === 'presetList'} />
+        <backend-ai-resource-preset-list active={curTabKey === 'preset'} />
       </Flex>
 
       <Flex
         style={{
-          display: curTabKey === 'registryList' ? 'block' : 'none',
+          display: curTabKey === 'registry' ? 'block' : 'none',
           height: 'calc(100vh - 145px)',
           // height: 'calc(100vh - 175px)',
         }}
       >
         {isSupportContainerRegistryGraphQL ? (
-          curTabKey === 'registryList' ? (
```

Comment:
I didn't work on this, so I don't know the exact details, but I know that if you set the display attribute to none, components will be rendered. I think it was probably used to prevent the `ContainerRegistryList` from rendering. I think we need to know the history. @yomybaby @gahyuun